imports/plugins/core/navigation/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,62 @@
+extend type Query {
+  "Get the navigation items for a shop"
+  navItems(shopId: ID!): NavItem
+}
+
+type NavItem {
+  "The Navigation Item ID"
+  _id: ID!
+
+  "The item's content. Can be plain text or HTML."
+  content: String!
+
+  "The link URL. Can be left empty to have context wrapped in a <span> instead of <a>."
+  url: String
+
+  "The ID of the parent navigation item, if applicable."
+  parentId: ID
+
+  "The order of the navigation item."
+  position: Int!
+
+  "The ID of the shop this navigation item belongs to."
+  shopId: ID!
+
+  "CSS styles to be added to the <a>/<span> tag"
+  styles: String
```

Comment:
I vote to remove `styles`. If someone really wanted this ability, they could do something custom putting styles on `metadata`. But we should not encourage this and should let the client be solely in control of its theme.

Reply:
Otherwise maybe a "styleVariant" string, which would have to match some string that the client UI is aware of. e.g., set it to "emphasized" and the client would make that a heavier weight font or something. Just as long as we're not injecting arbitrary CSS from the server config.